integration-tests/it-consumer/src/main/java/org/apache/servicecomb/it/testcase/TestDefaultValue.java

```diff
@@ -472,12 +524,21 @@ public void floatHeader_require_springmvc_rt() {
 
   @Test
   public void floatForm_require_springmvc_intf() {
-    assertEquals(defaultFloat, consumersSpringmvc.getIntf().floatFormRequire(null), 0.0f);
+    try {
+      consumersSpringmvc.getIntf().floatFormRequire(null);
+      assertEquals("required is true, throw exception", "but not throw exception");
+    } catch (Exception e) {
+      assertEquals(true, e.getMessage().contains("Parameter is not valid"));
+    }
   }
 
   @Test
   public void floatForm_require_springmvc_rt() {
-    assertEquals(defaultFloat,
-        consumersSpringmvc.getSCBRestTemplate().postForObject("/floatFormRequire", null, float.class), 0.0f);
+    try {
+      consumersSpringmvc.getSCBRestTemplate().postForObject("/floatFormRequire", null, float.class);
```

Comment:
Is this behavior change? We need to notice in RN.

Reply:
yes